server/src/test/java/org/opensearch/index/query/IntervalQueryBuilderTests.java

```diff
@@ -845,8 +845,8 @@ public void testWildcard() throws IOException {
         });
     }
 
-    private static IntervalsSource buildRegexpSource(String pattern, int flags, Integer maxExpansions) {
```

Comment:
Let's keep this as an overloaded method that calls:

```
buildRegexpSource(pattern, flags, 0, maxExpansions)
```
I'm not a fan of the C style flags that lucene uses here because it's hard to follow (I always have to look at the lucene java docs :/ )